eventsourcing/src/test/java/org/axonframework/eventsourcing/AnnotationBasedEventStateApplierTest.java

```diff
@@ -183,7 +183,6 @@ void invokesOnlyMostSpecificHandler() {
         }
     }
 
-    @Disabled("TODO #3313 - Event Sourcing Handler support for evolved state return")
```

Comment:
works!

Reply:
Again, amazing! Exactly what we wanted :-)